environment/environment.go

```diff
@@ -1018,3 +1022,15 @@ func DefaultJobLogFunction(e *Environment, message string) {
 		e.Cfg.fundReturnFailed = true
 	}
 }
+
+// addSafeToEvictPrevention adds the safe to evict annotation to the provided map if needed
+func addSafeToEvictPrevention(preventPodEviction bool, m map[string]string) map[string]string {
```

Comment:
Just wondering whether the method name gives an idea that we are adding it as safe-to-evict as true. Something like this might make it more clear. wdyt?
```suggestion
func markNotSafeToEvictPrevention(preventPodEviction bool, m map[string]string) map[string]string {
```

Reply:
Updated to markNotSafeToEvict